clients/pkg/promtail/client/client.go

```diff
@@ -39,28 +39,35 @@ const (
 	ClientLabel     = "client"
 	TenantLabel     = "tenant"
 	DropReasonLabel = "reason"
+	MutationLabel   = "mutation"
 
 	DropReasonGeneric             = "ingester_error"
 	DropReasonRateLimited         = "rate_limited"
 	DropReasonStreamLimited       = "stream_limited"
 	DropReasongMaxLineSizeLimited = "max_line_size_limited"
+
+	MutationTruncated = "truncated"
 )
 
 var DropReasons = []string{DropReasonGeneric, DropReasonRateLimited, DropReasonStreamLimited}
 
+var Mutations = []string{MutationTruncated}
+
 var UserAgent = fmt.Sprintf("promtail/%s", build.Version)
 
 type Metrics struct {
-	encodedBytes                 *prometheus.CounterVec
-	sentBytes                    *prometheus.CounterVec
-	droppedBytes                 *prometheus.CounterVec
-	sentEntries                  *prometheus.CounterVec
-	droppedEntries               *prometheus.CounterVec
-	requestDuration              *prometheus.HistogramVec
-	batchRetries                 *prometheus.CounterVec
-	countersWithHost             []*prometheus.CounterVec
-	countersWithHostTenant       []*prometheus.CounterVec
-	countersWithHostTenantReason []*prometheus.CounterVec
+	encodedBytes                   *prometheus.CounterVec
+	sentBytes                      *prometheus.CounterVec
+	droppedBytes                   *prometheus.CounterVec
+	sentEntries                    *prometheus.CounterVec
+	droppedEntries                 *prometheus.CounterVec
+	mutatedEntries                 *prometheus.CounterVec
+	requestDuration                *prometheus.HistogramVec
+	batchRetries                   *prometheus.CounterVec
+	countersWithHost               []*prometheus.CounterVec
+	countersWithHostTenant         []*prometheus.CounterVec
+	countersWithHostTenantReason   []*prometheus.CounterVec
+	countersWithHostTenantMutation []*prometheus.CounterVec
```

Comment:
We also don't need this `countersWithHostTenantMutation`. Just reuse `countersWithHostTenantReason`.